src/win/atomicops-inl.h

```diff
@@ -34,11 +32,16 @@
  * and interlocked operations on larger targets might require the target to be
  * aligned. */
 #pragma intrinsic(_InterlockedOr8)
+#pragma intrinsic(_InterlockedExchangeAdd)
 
 static char INLINE uv__atomic_exchange_set(char volatile* target) {
   return _InterlockedOr8(target, 1);
 }
 
+static unsigned INLINE uv__atomic_fetch_add(unsigned volatile* target, int increment) {
+  return (int)_InterlockedExchangeAdd((volatile LONG*)target, (LONG)increment);
```

Comment:
This does the wrong thing when `sizeof(LONG) != sizeof(int)`. Granted, they're normally equivalent but it still makes me a little uneasy, it's a landmine just waiting to go off.

Reply:
The inputs are `int`s that I extend to `long` for the operation and then back to `int` to return the value - this can backfire only if `int` is larger than `long` - which normally should not exist?
(however the `int` should be `unsigned`)

Reply:
That's what I mean by "landmine waiting to go off": it's not wrong now but it will be wrong in a non-LLP64 environment.

I was thinking of Windows-on-ARM64 specifically but maybe that's also LLP64. Quick googling was inconclusive.

At any rate, I'd be happier if this somehow failed to compile if the assumption turns out to be wrong. There's a STATIC_ASSERT macro you can use to enforce `sizeof(LONG) == sizeof(int)`.